### PR TITLE
Add Tests for TestPlanAdmin

### PR DESCRIPTION
**Pull request by man4oni**

### CODE REVIEW

File: tcms/testplans/tests/test_admin.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+class TestTestPlanAdmin(LoggedInTestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        initiate_user_with_default_setups(cls.tester)
+        cls.test_plan = TestPlanFactory()
+
+    def test_regular_user_can_not_add_testplan(self):
```
Comment (atodorov) [inline tcms/testplans/tests/test_admin.py:15]: Naming! It doesn't matter what level of permissions the user has, nobody can access the Add page. Same for change.

**Comment by zahari** [comment]
This is really good. From the get go to get encouraging words of Alex. You
are making headway!

File: tcms/testplans/tests/test_admin.py (hunk 1 above)
Comment (atodorov) [reply tcms/testplans/tests/test_admin.py:15]: Why did you fix this and then undo it ?

**Comment by man4oni** [comment]
I thought it was not correct

**Pull request merged**